vyper/types/convert.py

```diff
@@ -105,6 +106,17 @@ def to_int128(expr, args, kwargs, context):
                 pos=getpos(expr)
             )
 
+    elif input_type == 'address':
+        return LLLnode.from_list(
+            [
+                'signextend',
+                15,
+                ['and', in_arg, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF],
```

Comment:
I think using SizeLimits here would be cleaner?

Reply:
SizeLimits.ADDRSIZE specifically :)

Reply:
@jacqueswww ah good call! 😅 I had overlooked that that constant existed — will make this change tonight.